sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -1487,29 +1487,44 @@ class GpuRunningWindowIterator(
     val fixers = fixerIndexMap
     val numRows = cb.numRows()
 
+    var newOrder: Option[Array[Scalar]] = None
+
     withResource(computeBasicWindow(cb)) { basic =>
```

Comment:
I noticed this before when doing a run down of the code. computeBasicWindow will convert cb into a SpillableColumanrBatch.  I am not 100% sure why that works without any of the current changes, but I think we want to make it into a SpillableColumanrBatch before we pass it into computeBasicWindow.  So at least we are not doing it twice.